library/Translate/src/Juvix/Contextify.hs

```diff
@@ -37,14 +37,14 @@ op names = do
   case context of
     Left err -> pure $ Left err
     Right ctx -> do
-      (newCtx, _) <- Environment.runMIO (Passes.resolveModule ctx)
+      (newCtx, _) <- -- only Passes.resolveModule requires IO in these passes
+        Environment.runMIO $
+          Passes.resolveModule ctx
+            >>= Passes.inifixSoloPass
+            >>= Passes.recordPi
```

Comment:
Should we mention somehow which are `runM` or `runMIO`? 

It becomes more complciated, but it's good to note which need IO or not